reg/drone/srv/air_data_computer/_.0.1.uavcan

```diff
@@ -0,0 +1,48 @@
+# A generic air data computer (ADC) service.
+#
+# An air data computer service is generally a non-interactive publish-only service: when activated,
+# it keeps publishing its measurements at a fixed rate over several subjects until deactivated.
+# The activation/deactivation, if supported, is managed via the standard arming command service.
```

Comment:
Using an arming command here doesn't make much sense to me, and I am sure it will generated confusion between adopters. You usually associate arming to ESC/motors/servos, so actuators, not sensors, at least in the avionics world. We might need a parallel `activation` command service for these specific cases.

Reply:
Observe that we discussed the arming service as a generic entity that is applicable beyond actuators in the original PR. Let me add screenshots here for context because links to conversations work poorly on GitHub:

<img alt="image" src="https://user-images.githubusercontent.com/3298404/96717124-af3a4180-13ae-11eb-8ec0-d06a0122858d.png">

<img alt="image" src="https://user-images.githubusercontent.com/3298404/96717175-bd885d80-13ae-11eb-8186-c3d577cb1c3e.png">

I think the old PR showed that the concept of arming applied to other types of equipment does not cause confusion. I am keen on keeping some generic means of representing the degree of readiness but I am largely indifferent to its naming. If we perceive that arming is a poor choice of word, we can search for a new one although it's probably not worth the effort.

Reply:
What I am saying is that the concept of arm is something that is quite intrinsic to what you do in actuators. Not sensors. So the wording IMHO should not be applied the same way for both cases.

Reply:
I am going to move this question to the forum and if necessary we will address it in a later PR.

Reply:
https://forum.uavcan.org/t/service-readiness-level/967